metadata-ingestion/setup.py

```diff
@@ -75,7 +75,8 @@
     # With the release of 2.8.1, confluent-kafka only released a source distribution,
     # and no prebuilt wheels.
     # See https://github.com/confluentinc/confluent-kafka-python/issues/1927
-    "confluent_kafka[schemaregistry,avro]>=1.9.0, != 2.8.1",
+    # RegisteredSchema#guid is being used and was introduced in 2.10.1 https://github.com/confluentinc/confluent-kafka-python/pull/1978
+    "confluent_kafka[schemaregistry,avro]>=1.9.0, != 2.8.1, >=2.10.1",
```

Comment:
We can remove the old lower bound stuff. Can keep the comments for history if we want but the actual dep doesn’t need to stay

Reply:
indeed
```suggestion
    "confluent_kafka[schemaregistry,avro]>=2.10.1",
```